packages/pipeline/src/Pipeline.story.tsx

```diff
@@ -16,6 +18,9 @@ export default {
     children: {
       control: false,
     },
+    darkMode: {
+      control: 'boolean',
+    },
```

Comment:
Using `darkMode: storybookArgTypes.darkMode` will also give it the default comment I believe!